node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I think this file should be deleted, doesn't seem to be part of the webapp itself.